checks/oracle_jobs

```diff
@@ -79,8 +81,16 @@ def check_oracle_jobs(item, params, info):
     for line in info:
         service_found = False
 
-        if line[0] == sid:
-            data_found = True
+        if len(line) < 2:
+            # ignore wrong/corrupted lines
+            continue
+        else:
+            if line[1].startswith(' Debug '):
```

Comment:
Does the corrupt entry always start with a space?
More robust: line[1].strip().startswith("Debug")

Reply:
> Does the corrupt entry always start with a space?
> More robust: line[1].strip().startswith("Debug")

Yes, because it is imposible that an identifier in whole mk_oracle could start with a ' '.
=> We are sure that this is really a debug informatione, whenn ' Debug ' is found in the item.
The whole '|Failure|' in mk_oracle is a broken by design from very old days. That's why I change it to Debug instead going on with the broken design.